### PR TITLE
Update Katib doc for Sobol's quasirandom sequence support

### PR DESCRIPTION
**Pull request by c-bata**
Update Katib documentation for Sobol's quasirandom sequence support (refs https://github.com/kubeflow/katib/pull/1523).

cc: @andreyvelich

### CODE REVIEW

File: content/en/docs/components/katib/experiment.md [hunk 1]
```diff
@@ -436,6 +437,21 @@ Katib supports the following algorithm settings:
...
+The algorithm name in Katib is `sobol`.
+
+Katib uses the [Goptuna](https://github.com/c-bata/goptuna) optimization
+framework for its Sobol's quasirandom search.
+
+The [Sobol's quasirandom sequence](https://dl.acm.org/doi/10.1145/641876.641879)
+is a low-discrepancy sequence. And it is known that Sobol's quasirandom sequence can
+provide better uniformity properties.
+
+Currently, Katib does not support no algorithm settings for Sobol.
```
Comment (andreyvelich) [inline content/en/docs/components/katib/experiment.md:453]: I think we can remove this notification similar to Grid: https://www.kubeflow.org/docs/components/katib/experiment/#grid-search.

```suggestion
```

**Review by andreyvelich — COMMENTED** [review]
Sorry for the late review @c-bata.
I left one comment.

**Review by c-bata — COMMENTED** [review]
Apply a suggestion from review.

**Review by andreyvelich — APPROVED** [review]
Thank you for implementing this @c-bata!
/lgtm
/approve

**Pull request merged**